recipes/opengv/all/test_package/conanfile.py

```diff
@@ -1,17 +1,26 @@
-from conans import ConanFile, CMake, tools
 import os
-
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.cmake import CMake
+from conan.tools.layout import cmake_layout
 
 class TestPackageConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
-    generators = "cmake", "cmake_find_package_multi"
+    generators = "CMakeDeps", "CMakeToolchain"
```

Comment:
```suggestion
    generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
```
It injects all environment variables